docs/csharp/quick-starts/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Quick Starts - C# Guide | Microsoft Docs
+description: Learn C# in your browser
+keywords: C#, Get Started, Lessons, Interactive
+author: billwagner
+ms.author: wiwagn
+ms.date: 09/18s/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# C# Quick starts #
+
+Welcome to the C# Quick starts. These start with interactive lessons
+that you can run in your browser.
+
+The first lessons explain C# concepts using small snippets of code. You'll
+learn the basics of C# syntax and how to work with data types like strings,
+numbers, and booleans. It's all interactive, and you'll be writing code
+within minutes. These first lessons assume no prior knowledge of
+programming or the C# language.
+
+## [Hello world](hello-world)
```

Comment:
the link works but OPS doesn't like this. Usually we need to add the extension too. Not sure how this is done with quick-starts. Do we need to add yml to the link so OPS doesn't give us a warning? /cc @eisenbergeffect